Makefile

```diff
@@ -53,8 +53,9 @@ shell: build-container
 
 check: validate test-unit test-integration
 
+# The tests can run out of entropy and block in containers, so replace /dev/random.
 test-integration: build-container
-	$(DOCKER_RUN_DOCKER) hack/make.sh test-integration
+	$(DOCKER_RUN_DOCKER) bash -c 'rm -f /dev/random; ln -sf /dev/urandom /dev/random; hack/make.sh test-integration'
```

Comment:
See the comment just above :)

There is apparently no way (or no reasonable way?) to convince GPG not to use `/dev/random`, and in the Travis environment we don't have write access to the RNG so we can’t even send fake entropy in there.

But the container is all ours to play with, so we can point `/dev/random` to the non-blocking `urandom` device, which fixes the issue and does not affect the surrounding environment.

Does this need a better comment?